### PR TITLE
change to vertical mobile layout & disable image drag

### PR DESCRIPTION
**Pull request by timothy-liang**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -91,19 +97,19 @@ body {
 }
 
 .stack {
-  height: 100%;
-  min-width: 50%;
+  width: 40%;
+  min-width: 150px;
+  /* adjust for the asset's asymmetric crop */
+  margin-right: -4.119vw;
```
Comment (cktang88) [inline src/index.css:103]: this is a really weird value

Comment (timothy-liang) [reply src/index.css:103]: I just moved it over from a previous place, was this calculated somehow @k-zhai?

Comment (k-zhai) [reply src/index.css:103]: Yeah it was calculated

**Comment by cktang88** [comment]
so are we gonna merge this?

**Pull request closed without merging**